subworkflows/nf-core/utils_nfcore_pipeline/tests/main.function.nf.test

```diff
@@ -100,15 +98,14 @@ nextflow_function {
 
 
 
-    test("Test Function logColours") {
+    test("Test Function without logColours") {
 
         function "logColours"
 
         when {
             function {
                 """
-                // define inputs of the function here. Example:
-                // input[0] = 1
+                // input[0] = false
```

Comment:
logColours arg `monochrome_logs` value needs to be uncommented. I think to match the test case descriptions you also need to flip the boolean- "true" uses monochrome (== no log colour). Actually for this one you could leave this commented because `monochrome_logs=true` is the default
```suggestion
                input[0] = true
```